fitbenchmarking/controllers/ralfit_controller.py

```diff
@@ -69,6 +75,18 @@ def setup(self):
         elif self.minimizer == "hybrid_reg":
             self._options[b"model"] = 3
             self._options[b"type_of_method"] = 2
+        elif self.minimizer == "newton":
+            self._options[b"model"] = 2
+            self._options[b"type_of_method"] = 1
+        elif self.minimizer == "newton_reg":
+            self._options[b"model"] = 2
+            self._options[b"type_of_method"] = 2
+        elif self.minimizer == "newton-tensor":
+            self._options[b"model"] = 4
+            self._options[b"type_of_method"] = 1
+        elif self.minimizer == "newton-tensor_reg":
+            self._options[b"model"] = 4
+            self._options[b"type_of_method"] = 2
```

Comment:
I've found what the numbers mean on the RALFit documentation, these all look fine